experiments/calibration/global_bucket/calibrate_bucket_function.jl

```diff
@@ -0,0 +1,432 @@
+function bucket_turbulent_fluxes(params)
+    t0 = 0.0
+    tf = 60 * 60.0 * 24 * 365
+    Δt = 900.0
+    nelements = (101, 7)
+    regridder_type = :InterpolationsRegridder
+    radius = FT(6378.1e3)
+    depth = FT(3.5)
+    diagnostics_outdir = joinpath(root_path, "global_diagnostics")
+    outdir = ClimaUtilities.OutputPathGenerator.generate_output_path(
+        diagnostics_outdir,
+    )
+    prob, cb = setup_prob(t0, tf, Δt, params, outdir; nelements)
+
+    timestepper = CTS.RK4()
+    ode_algo = CTS.ExplicitAlgorithm(timestepper)
+    SciMLBase.solve(prob, ode_algo; dt = Δt, callback = cb, adaptive = false)
+
+    simdir = ClimaAnalysis.SimDir(
+        joinpath(root_path, "global_diagnostics", "output_active"),
+    )
+    lhf = get(simdir; short_name = "lhf")
+    shf = get(simdir; short_name = "shf")
+
+    domain = ClimaLand.Domains.SphericalShell(;
+        radius = radius,
+        depth = depth,
+        nelements = nelements,
+        npolynomial = 1,
+        dz_tuple = FT.((1.0, 0.05)),
+    )
+    surface_space = domain.space.surface
+
+    # Define the locations longitudes and latitudes
+    training_locations, validation_locations =
+        rand_locations(surface_space, regridder_type, 5)
+
+    # Read in the era5 datafile
+    era5_ds = Dataset(joinpath(ClimaLand.Artifacts.era5_surface_data2008_path(),
+                               "era5_monthly_surface_fluxes_200801-200812.nc"))
+
+    # Initialize an empty list to store observations
+    obs_list = []
+    i = 0
+    # Loop over each location
+    # for (lon, lat) in training_locations
+    #     i += 1
+    #     # Slice lhf and shf at the given longitude and latitude
+    #     lhf_loc = ClimaAnalysis.slice(lhf, lon = lon, lat = lat)
+    #     shf_loc = ClimaAnalysis.slice(shf, lon = lon, lat = lat)
+
+    #     # Create Observation objects for lhf and shf
+    #     lhf_obs = EKP.Observation(
+    #         Dict(
+    #             "samples" => lhf_loc.data,
+    #             "covariances" => cov(lhf_loc.data) * EKP.I,
+    #             "names" => "lhf_$(lon)_$(lat)",
+    #         ),
+    #     )
+    #     shf_obs = EKP.Observation(
+    #         Dict(
+    #             "samples" => shf_loc.data,
+    #             "covariances" => cov(shf_loc.data) * EKP.I,
+    #             "names" => "shf_$(lon)_$(lat)",
+    #         ),
+    #     )
+
+    #     # Add the observations to the list
+    #     push!(obs_list, lhf_obs)
+    #     push!(obs_list, shf_obs)
+    # end
+
+    close = (x, y) -> abs(x - y) < 5e-1
+    for (lon, lat) in training_locations
+        i += 1
+        # Fetch slices of lhf and shf era5 data from the era5 dataset
+        lat_ind, lon_ind = findall((x) -> close(x, lat), era5_ds["latitude"][:])[1],
+                           findall((x) -> close(x, lon + 180), era5_ds["longitude"][:])[1]
+        lhf_loc = vec(era5_ds["mslhf"][lon_ind, lat_ind, :][:, end, :])
+        shf_loc = vec(era5_ds["msshf"][lon_ind, lat_ind, :][:, end, :])
```

Comment:
Using this version of the loop I was able to successfully run the job_script on central/GPU with era5 lhf/shf data.

Reply:
(Using 5 randomly sample (lon, lat))

Unfortunately, we don't output the lat/lon that we use anywhere that I can tell, so I'm, not actually sure which locations it was running on.

Reply:
@Espeer5 

I believe when running the job you were still calibrating the perfect model which was the target?

I now changed the target to ERA5, but I get the error you mention:

```
era5_surface_data2008_path not defined in ClimaLand.Artifacts
```

Reply:
Hmmm this seems strange to me - this would imply it's not able to find the function definition within the Artifacts module? But the diff for this PR shows it should be there?

Reply:
I rebased and changed the Manifests, maybe that's why?